frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -162,12 +163,16 @@ export class ReduxTlmDispatcher extends React.Component {
     this.handleNotification(data)
   }
 
-  handleMemberCreated = data => {
+  handleMemberCreated = async data => {
     const { props } = this
 
     if (props.user.userId === data.fields.user.user_id) {
       props.dispatch(addRoleWorkspaceList(data.fields.user, data.fields.member, data.fields.workspace))
       props.dispatch(removeAccessibleWorkspace(data.fields.workspace))
+      const fetchGetKnownMemberList = await props.dispatch(getMyselfAllKnownMember())
+      if (fetchGetKnownMemberList.status === 200) {
+        props.dispatch(setKnownMemberList(fetchGetKnownMemberList.json))
+      }
```

Comment:
Couldn't this whole bit be put as a `then().catch()` block instead to not make the handler asynchronous ?